OData/src/System.Web.OData/OData/EnableQueryAttribute.cs

```diff
@@ -534,16 +571,17 @@ private object ExecuteQuery(object response, HttpRequestMessage request, HttpAct
             Type elementClrType = GetElementType(response, actionDescriptor);
 
             IEdmModel model = GetModel(elementClrType, request, actionDescriptor);
-            if (model == null)
-            {
+            if (model == null) {
```

Comment:
I will modify after I understand the rest of the comments so I will modify everything at once.